contentcuration/contentcuration/models.py

```diff
@@ -429,32 +480,22 @@ def save(self, *args, **kwargs):
                 original.parent.changed = True
                 original.parent.save()
 
-        super(ContentNode, self).save(*args, **kwargs)
-
-        post_save_changes = False
         if self.original_node is None:
             self.original_node = self
-            post_save_changes = True
         if self.cloned_source is None:
             self.cloned_source = self
-            post_save_changes = True
 
         if self.original_channel_id is None and self.get_channel():
```

Comment:
This will make the channel null as the node isn't part of the tree until saved. Might be better to move this logic to after the super.save